stylesheets/main.css

```diff
@@ -470,6 +470,26 @@ ul.feature-links > li {
 .feature-anchor {
   color: #666;
 }
+
+.ui5-sticker {
+  display:block;
+  float:right;
+  margin-top:40px;
+  margin-right:10%;
+  z-index:999;
+  position: relative;
+}
+.bangalore-flag {
+  position:absolute;
+  bottom: 0;
+  right: 0;
+}
+
```

Comment:
please add prefix "ui5-sticker__" to the both css classes.
I would recommend to group all css classes related to the ui5con sticker in on area (e.g. in the end of the file) and add a comment before it as it done for other areas:
/***********************************
UI5con sticker
/***********************************

Reply:
I didn't get  (add prefix "ui5-sticker__" to the both css classes)
I deleted one class. It was created by me and has the same name as we did together